commons/clever-tools/ssh-access.md

```diff
@@ -0,0 +1,64 @@
+---
+title: SSH access
+shortdesc: SSH access to running instances
+tags:
+- cli-setup
```

Comment:
The apps tag would be fine, don't you think? (https://www.clever-cloud.com/doc/apps/)